scripts/create-symlinks.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Usage in the project root folder:
+# npm run init -- PACKAGE_NAME
+
+
+if [ -z "$1" ]
+	then
+		echo "Please provide a package name!";
+		exit 1;
+fi
+
+packageDir="./packages/$1";
+# Just in case package folder doesn't exist yet.
+mkdir -p $packageDir;
+
+files=(".prettierrc.json" ".prettierignore" ".npmrc" ".nycrc" ".npmignore" ".babelrc" ".eslintignore" "cypress.json" "cypress" "browsertest" "scripts")
+
+
+for i in "${files[@]}"
+do
+	if [ ! -e "$packageDir/${i}" ]
+		then
+			ln -s ../../templates/$i.tmpl $packageDir/$i;
+			printf "Symlink Created: $packageDir/$i -> ../../templates/$i.tmpl\n";
+	fi;
+done
+
+chmod +x $packageDir/scripts/*.sh
```

Comment:
👍

Reply:
Why not make the scripts executable in git?